### PR TITLE
Add number of videos on discovery screen as an option.

### PR DESCRIPTION
**Pull request by cruzanmo**

### CODE REVIEW

File: js/skin.js [hunk 1]
```diff
@@ -241,7 +241,7 @@ var Skin = React.createClass({
               icon="discovery">
               <DiscoveryPanel
                 {...this.props}
-                videosPerPage={{xs:2, sm:4, md:6, lg:8}}
+                videosPerPage={this.state.config.nym && this.state.config.nym.discoveryVideosPerSlide || {xs:2, sm:4, md:6, lg:8}}
```
Comment (byronhulcher) [inline js/skin.js]: what would happen if you didn't provide a value? like if you left out `md: 6`?

Comment (cruzanmo) [inline js/skin.js]: Good point.

Comment (byronhulcher) [inline js/skin.js]: Maybe something like `videosPerPage = _.assign({xs:2, sm:4, md:6, lg:8}, this.state.config.nym ? this.state.config.nym.discoveryVideosPerSlide : {} })`

**Comment by byronhulcher** [comment]
👍

**Pull request merged**